test/unit/scripts/upgrade/TimeConstraints.t.sol

```diff
@@ -34,97 +34,182 @@ contract TimeConstraintsTest is Test {
         assertEq(timeConstraints.getCurrentDayTime().toSeconds(), 0);
     }
 
-    function test_checkExecuteWithinDayTime_HappyPath() external {
-        vm.warp(BASE_TIMESTAMP + 15 hours);
-        timeConstraints.checkExecuteWithinDayTime(Durations.from(12 hours), Durations.from(18 hours));
-
-        vm.warp(BASE_TIMESTAMP + 12 hours);
-        timeConstraints.checkExecuteWithinDayTime(Durations.from(12 hours), Durations.from(18 hours));
-
-        vm.warp(BASE_TIMESTAMP + 18 hours);
-        timeConstraints.checkExecuteWithinDayTime(Durations.from(12 hours), Durations.from(18 hours));
+    // checkTimeWithinDayTime
+
+    function testFuzz_checkTimeWithinDayTime_HappyPath_Regular(
```

Comment:
This test narrows the chain time to the range [0, 24 hours]. Update the test to allow the full `uint40` range for the chain time, for example:
```solidity
    function testFuzz_checkTimeWithinDayTime_HappyPath_Regular(uint32 from, uint32 to, uint40 timeToWarp) external {
        from = from % 24 hours;
        to = to % 24 hours;
        uint256 warpDayTime = timeToWarp % 24 hours;

        vm.assume(from <= to && warpDayTime >= from && warpDayTime <= to);

        vm.warp(timeToWarp);
        timeConstraints.checkTimeWithinDayTime(Durations.from(from), Durations.from(to));
    }
```

Reply:
Use such an approach accross all tests instead of adding time to `BASE_TIMESTAMP`